### PR TITLE
bwa_v0.6.2

### PR DESCRIPTION
**Pull request by JinfengChen**
* [ y] I have read the guidelines above.
* [ y] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/bwa/0.6.2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package:
+  name: bwa
+  version: "0.6.2"
+
+build:
+  number: 0
+
+source:
+  fn: bwa-0.6.2.tar.bz2
+  url: http://downloads.sourceforge.net/project/bio-bwa/bwa-0.6.2.tar.bz2?r=https%3A%2F%2Fsourceforge.net%2Fprojects%2Fbio-bwa%2Ffiles%2F&ts=1478640973&use_mirror=iweb
```
Comment (bgruening) [inline recipes/bwa/0.6.2/meta.yaml]: Can you please add a checksum here.

Comment (bgruening) [inline recipes/bwa/0.6.2/meta.yaml]: The URL can also be simlified, after the ?r=

**Comment by JinfengChen** [comment]
updated. Now I have two commits to merge. Is that okay?
Thanks.

Jinfeng

**Comment by JinfengChen** [comment]
Hi,
Can you give me some clue on why the job failed on SUBDAG=0 but passed for SUBDAG=1? I do not know the difference between these two. 
Thanks.
Jinfeng

**Comment by bgruening** [comment]
If you submit many packages at once we try to split them into 2 parts. If you submit only one recipes, one process (subdag) is doing nothing.

File: recipes/bwa/0.6.2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  fn: bwa-0.6.2.tar.bz2
+  url: http://downloads.sourceforge.net/project/bio-bwa/bwa-0.6.2.tar.bz2
+  md5: 5e4043769ce3436f6098ec67f6e02332
+
+requirements:
+  build:
+    - gcc
+    - zlib
+  run:
+    - gcc
```
Comment (bgruening) [inline recipes/bwa/0.6.2/meta.yaml]: gcc is not needed as run time dependency, do you mean libgcc

**Comment by JinfengChen** [comment]
Hi, @bgruening 

How long need to wait until the passed commits to be merged with master branch and available from bioconda-recipes? I am testing another package that depended on bwa-0.6.2. I need to have a timeline when I can move forward. Thanks.

Jinfeng

**Comment by bgruening** [comment]
Usually, we do this really fast. But I was travelling, sorry missed that.
If you want I can add you as contributor and you can merge your own update PRs if you stick to the Guidelines.

**Comment by JinfengChen** [comment]
Yes, that would be great.
Thanks.

Jinfeng

2016年11月9日星期三，Björn Grüning <notifications@github.com> 写道：

**Pull request merged**